### PR TITLE
Update description of project in the README

### PR DESCRIPTION
**Pull request by chrismytton**
This is a copy/paste of the [wording that was in the data_pr_change_summarizer README](https://github.com/everypolitician/data_pr_change_summarizer/blob/1521f1cc257122d57ee076c17d89a003810ab414/README.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
 # EveryPolitician Pull Request summary
 
-Produces a summary of an EveryPolitician data pull request like [this one](https://github.com/everypolitician/everypolitician-data/pull/16984#issuecomment-246282991).
+This is the code the EverPolitician bot uses to inspect data pull requests (specifically those are on the
```
Comment (tmtmtmtm) [inline README.md]: "those are on" seems wrong here. Probably should either be "those that are on" or just "those on"

File: README.md [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
+[everypolitician-data](https://github.com/everypolitician/everypolitician-data) repo)
+and add helpful summaries to them. The bot writes the summary as a comments on the Pull Request
```
Comment (tmtmtmtm) [inline README.md]: "a comments"?

File: README.md (hunk 1 above)
Comment (chrismytton) [reply README.md]: Good catch! Fixed in https://github.com/everypolitician/everypolitician-pull_request/pull/8/commits/cac2fffbca396b9b45952f173756565e1f04dc77 (and I fixed the "EverPolitician" typo as well 😉 )

File: README.md (hunk 2 above)
Comment (chrismytton) [reply README.md]: Yikes! Fixed in https://github.com/everypolitician/everypolitician-pull_request/pull/8/commits/cac2fffbca396b9b45952f173756565e1f04dc77.

**Pull request merged**